hedera-node/src/main/java/com/hedera/services/context/SideEffectsTracker.java

```diff
@@ -435,6 +459,8 @@ public void reset() {
 		newAccountId = null;
 		newContractId = null;
 		newEntityAlias = ByteString.EMPTY;
+		pseudorandomNumber = 0;
```

Comment:
Since `0` is a valid output of the PRNG, I'd lean toward using `-1` as a sentinel value here.

Reply:
Fixed . Thanks Michael !